### DISCUSSION TITLE
Access to GitHub via Chrome

### DISCUSSION THREAD

**Discussion by miQSH** [community/community#53197] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am using Opera and Chrome to sign on to GitHub. Opera prompts me to consult my 2fa, ok. But Chrome every time insists on a new backup code. While I have them, it seems unnatural since I haven't lost access to my account.

How can I change the behavior of Chrome?
Thank you.

**Comment by neutrinomuon** [comment, marked as answer]
Hello @miQSH,

Indeed, it seems a little bit odd. it looks like Chrome might not be remembering your login credentials... Including your 2FA codes. Maybe, you can try the following check-ups:

- Enable Chrome to save passwords: Can you check in Chrome Settings > Autofill > Passwords, and make sure that the "Offer to save passwords" option is turned on?

- Clear data (saved passwords, etc...): In case the previous part does not work, can you go to Chrome Settings > Privacy and security > Clear browsing data? You can select "All time" (BUT BE CAREFUL, it is a bit annoying 😔 since it erases everything), make sure that "Passwords" and "Site data" are checked;

- Log in to GitHub using Chrome and check the "Remember me" option;

- Enter your 2FA backup code when asked, and check the option to trust the device.

I also saw that If you're still asked to provide a backup code every time you log in, even with following the previous advice, it might be that some extension could be interfering with your login process... This could be potentially more difficult to find out, but you may try to disable the browser extensions, or try using incognito mode to see if the problems persist.

I hope this helps. Please, let me know if it works.

**Reply by miQSH** [reply]
Thank you for your helpful reply. It worked :)

**Reply by neutrinomuon** [reply]
Hi @miQSH, I'm very glad to hear that it worked :). Have a great day!
